<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>726</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1173</width>
    <height>726</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1173</width>
    <height>726</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home Nutrizionista</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>726</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1173</width>
     <height>726</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>homeNutrizionista.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestDieta">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>GESTIONE DIETA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_visualizzaClienti">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>VISUALIZZA CLIENTI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestVisite">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>GESTIONE VISITE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_visualizzaTurni">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>VISUALIZZA TURNI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Nutrizionista.qrc"/>
 </resources>
 <connections/>
</ui>
